model/src/main/java/com/ge/predix/acs/rest/attribute/adapter/AttributesResponse.java

```diff
@@ -33,4 +39,28 @@ public String getId() {
     public void setId(final String id) {
         this.id = id;
     }
+
+    public int checkSizeLimit(final int maxNumberOfAttributes, final int maxSizeOfAttributesInBytes) {
```

Comment:
See comment above re. recomputing using local variables and removing private member variables.

Also maybe the following signature is more clear since what we really care about is whether or not we've exceeded set thresholds:

`public boolean sizeLimitsExceeded ...`

This (and related refactoring inside this method) would also allow us to get rid of the reliance on `Integer.MAX_VALUE`